src/doc/manifest.md

```diff
@@ -527,9 +527,13 @@ name = "..."
 crate-type = ["dylib"] # could be `staticlib` as well
 ```
 
-The available options are `dylib`, `rlib`, and `staticlib`. You should only use
-this option in a project. Cargo will always compile packages (dependencies)
-based on the requirements of the project that includes them.
+The available options are `dylib`, `rlib`, `staticlib`, and, as of Rust 1.11, 
```

Comment:
We don't generally make statements like "as of Rust version" in the docs, as the deployed ones are for the current version, and the right versions come with each version of the compiler.